drake/doc/code_style_guide.rst

```diff
@@ -47,12 +47,22 @@ There are several tools for helping you to compile with the required style. They
 ClangFormat
 ^^^^^^^^^^^
 
-(*Notes comming soon*)
+The following instructions were tested on OS X but should also work on Linux. A different installation procedure may need to be followed on Windows.
```

Comment:
On Ubuntu, clang-format is already packaged.  Ubuntu users should use the pre-packaged version, not a separate download.

Even on OSX, wouldn't it be better to use the macports version of clang-format?  Downloading and hand-installing software is hard to maintain; packaged versions are almost always better.